# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/client/TextureClient.h
# Commit: 220d6288fe81
# Full Hash: 220d6288fe8156df13e9582af79702ae5ef1f237
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2022-12-09 21:45:26
# Regressor Bug: 1795768
# File Overlap Count: 1
# Description:
#   Bug 1795768 - Do not use Direct2D canvas if willReadFrequently is set. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D164364
# ==============================================================================

diff -r 5d43ff0044fe -r 220d6288fe81 gfx/layers/client/TextureClient.h
--- a/gfx/layers/client/TextureClient.h	Fri Dec 09 17:03:03 2022 +0000
+++ b/gfx/layers/client/TextureClient.h	Fri Dec 09 17:03:03 2022 +0000
@@ -95,6 +95,9 @@
   // The texture is going to be updated using UpdateFromSurface and needs to
   // support that call.
   ALLOC_UPDATE_FROM_SURFACE = 1 << 7,
+
+  // Do not use an accelerated texture type.
+  ALLOC_DO_NOT_ACCELERATE = 1 << 8,
 };
 
 enum class BackendSelector { Content, Canvas };
